test/unit/double_free.c

```diff
@@ -21,6 +21,15 @@ test_double_free_post() {
 	safety_check_set_abort(NULL);
 }
 
+bool enable_tcache() {
+	bool tcache_enabled;
+	size_t sz = sizeof(tcache_enabled);
+	assert_d_eq(
+	    mallctl("thread.tcache.enabled", &tcache_enabled, &sz, NULL, 0), 0,
+	    "Unexpected mallctl failure");
+	return !tcache_enabled;
```

Comment:
Shouldn't this return `tcache_enabled` instead of `!tcache_enabled`?

Reply:
Yes, `return tcache_enabled` makes more sense to me. Modified and tweaked the logics when invoking as well.